### ISSUE TITLE
A11y_JupyterNotebookWidget_SummaryImportanceTab_Keyboard:Out[10] control present below screen is not accessible by keyboard.

### ISSUE THREAD

**Issue by nirmalsahoo** [jupyter/notebook#5373]
Environment Details:
Application Name: Jupyter Notebook Widget
Browser Details: Microsoft Edge 44.17763.1.0,Chrome Version 75.0.3770.142 (Official Build) (64-bit)
Operating System: Windows10

Repro Steps

1.Launch Jupyter Notebook from your local directory.
2.Login with the appropriate credentials to azure notebook account.
3.Navigate to Summary Importance tab.
4.Navigate through tab key to access the out[10].
5.Check whether the out[10] is accessible by keyboard present below the screen.

**Actual:**

While navigating through tab key, Out[10] control present below screen is not accessible by keyboard.

**Expected:**

While navigating through tab key, Out[10] control present below screen should be  accessible by keyboard.
**User Impact**

Keyboard user unable to access the coordinates points on the graph.

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
